test/unit/middlewares/impersonationRequests.test.ts

```
@@ -0,0 +1,44 @@
+import chai from "chai";
+import sinon from "sinon";
+import { getImpersonationRequestsValidator } from "../../../middlewares/validators/impersonationRequests";
+import { CreateImpersonationRequestBody } from "../../../types/impersonationRequest";
+
+const { expect } = chai;
+
+describe("Impersonation Request Validators", function () {
+  let req: any;
+  let res: any;
+  let requestBody: CreateImpersonationRequestBody = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'requestBody' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)